src/app/PCM/templates/tpl-sbe-row-add/sbe-add-element.js

```diff
@@ -17,7 +18,13 @@ const SBEAddElement = ({
 
   const component = askData?.question?.attribute?.dataType.component || ''
 
-  return equals(component)('dropdown') && !disabled ? (
+  if (component === '') {
```

Comment:
do a boolean coercion and check against that value for component, so that it also throws out the console warns in case of other falsy values such as null or undefined etc.

Reply:
Will do, I will correct that in the next push